.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I would prefer if you aligned with the eslint configuration used in Antora core. See https://gitlab.com/antora/antora/-/blob/master/.eslintrc